spec/features/user_updates_account_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'user can update account' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.